pages/rooms/[roomId].tsx

```diff
@@ -14,20 +14,32 @@ const RoomId = () => {
     dial_out: string;
   };
 
+  const [isRouterReady, setIsRouterReady] = useState(false);
+
   const optionalFeatures = {
     isNetworkMetricsEnabled: queryParameters.network_metrics === 'true',
     isSimulcastEnabled: queryParameters.simulcast === 'true',
     isDialOutEnabled: queryParameters.dial_out === 'true',
   };
 
+  useEffect(() => {
+    if (!router.isReady) {
+      return;
+    }
+
+    setIsRouterReady(true);
+  }, [router.isReady]);
+
   return (
     <Fragment>
-      <Rooms
-        id={queryParameters.roomId}
-        clientToken={queryParameters.client_token}
-        refreshToken={queryParameters.refresh_token}
-        optionalFeatures={optionalFeatures}
-      />
+      {isRouterReady && (
```

Comment:
Why don't use this `router.isReady` directly here?

Reply:
Because I must have been sleeping while I wrote it. 😄 
Resolved in [0b7cb1c](https://github.com/team-telnyx/telnyx-meet/pull/75/commits/0b7cb1c14d3875b7ab4cf0d9262527c4dfd88503).

Thanks, @DeividVeloso!